### ISSUE TITLE
tinyYoloV2 making my code collapse

### ISSUE THREAD

**Issue by sagilehman** [justadudewhohacks/face-api.js#628]
Hi,
I'm using your faceExpressionsNet and when i start my code it immediately collapses and shows the following error:
![image](https://user-images.githubusercontent.com/46101284/83390820-389aa880-a3fb-11ea-92fe-10dae221b687.png)
Do you have any idea what might solve it?

**Comment by Ulrira2k** [comment]
Sounds like you didn't load the model before actually trying to use it.

**Comment by sagilehman** [comment]
I did, it doesn't work

**Comment by Ulrira2k** [comment]
Yeah, I figured. Just saw another issue where somebody said he had loaded the model but got this error. That's why I deleted my comment again, seeming as that was probably not the reason ;)
